devices/shenzhen_homa.js

```diff
@@ -3,6 +3,60 @@ const extend = require('../lib/extend');
 const e = exposes.presets;
 
 module.exports = [
+    {
+        fingerprint: [
+            {modelID: 'HOMA1001', endpoints: [
+                {ID: 10, profileID: 49246, deviceID: 256, inputClusters: [0,3,4,5,6,8], outputClusters: []},
+                {ID: 11, profileID: 49246, deviceID: 528, inputClusters: [0,3,4,5,6,8,768], outputClusters: []},
+                {ID: 13, profileID: 49246, deviceID: 57694, inputClusters: [4096], outputClusters: [4096]},
+            ]},
+        ],
+        model: 'HOMA1001',
```

Comment:
The model should be unique, I suggest to postfix all the models with the functionallity, e.g. `HOMA1001_RGB`, `HOMA1001_CT`, etc.